packages/core/src/browser/fetchObservable.ts

```diff
@@ -113,5 +114,15 @@ function afterSend(
       observable.notify(context)
     }
   }
-  responsePromise.then(monitor(reportFetch), monitor(reportFetch))
+
+  const reportFetchOnPayloadComplete = (response: Response) => {
+    const cloneResponse = response.clone()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could we have a test ensuring that we are waiting for the response download to report the fetch?